Example/AztecUITests/HighPriorityIssuesTests.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  HighPriorityIssuesTests.swift
+//  AztecUITests
+//
+//  Created by brbrr on 11/10/17.
+//  Copyright © 2017 Automattic Inc. All rights reserved.
+//
+
+import XCTest
+
+class HighPriorityIssuesTests: XCTestCase {
+        
+    private var app: XCUIApplication!
+    private var richTextField: XCUIElement!
+    
+    override func setUp() {
+        super.setUp()
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIDevice.shared().orientation = .portrait
+        app = XCUIApplication()
+        app.launch()
+        
+        let tablesQuery = app.tables
+        tablesQuery.staticTexts[elementStringIDs.emptyDemo].tap()
+        
+        let richTextField = app.textViews[elementStringIDs.richTextField]
+        richTextField.tap()
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    // Gihtub issue #393
+    func testLongTitle() {
+        //    Title line height is about 22px, so it might be useing for comparing the height difference should make it precise.
+        //    But may be fragile due to different font sizes etc
+        let titleLineHeight = 22
+        
+        let titleTextView = app.textViews[elementStringIDs.titleTextField]
+        titleTextView.tap()
+        
+        let oneLineTitleHeight = Int(titleTextView.frame.height)
+        
+        titleTextView.typeText("very very very very very very long title in a galaxy not so far away")
+        let twoLineTitleHeight = Int(titleTextView.frame.height)
+        XCTAssert(twoLineTitleHeight - oneLineTitleHeight == titleLineHeight )
```

Comment:
Instead of setting `titleLineHeight` (fragile due to device settings or design changes) and using it in this calculation, what about using `oneLineTitleHeight` and asserting that `twoLineTitleHeight` is twice that? (Or just that `twoLineTitleHeight` is a multiple of `oneLineTitleHeight` if we want to confirm that the title area expands.)

Reply:
I totally agree with you and I was thinking about that one. The thing is that `oneLineTitleHeight` is about `38px` (due to underline and some padding), but every other line is `22px`. So its quite complicated to confirm that in a nice way. We can confirm that it's just bigger, but I don't think its good approach either.

Reply:
Ah right, I see what you mean. I agree that it'd be best to confirm a specific change in line height, so it might be best to stick with this approach. I'll keep thinking about it. (In the future, if we implement visual regression tests for Aztec we could confirm in the UI test that it's just bigger and use the visual regression test to make sure the entire title is visible. I wouldn't rely on that until we start implementing those visual tests, though.)